### DISCUSSION TITLE
Greetings

### DISCUSSION THREAD

**Discussion by MUGWANEZAMANZI** [community/community#146529] [GitHub Education] [answered]
### Hi everyone,

Hi My names are MUGWANEZA MANZI it is my pleasure to join the GitHub education.
Am doing bachelor's of Computer Science in year 2 and am passionate about making games and cyber stuffs

**Comment by jorritvanderheide** [comment, marked as answer]
Hi MUGWANEZA, if you mark my post as the answer to your question, I will follow your profile!

**Reply by MUGWANEZAMANZI** [reply]
Marked!
